packages/node-opentelemetry/src/sentry.ts

```diff
@@ -45,10 +57,13 @@ const startOtelSpanFromSentryEvent = (event: Event, hint: EventHint) => {
     ...(event.modules && {
       [SEMATTRS_EXCEPTION_MODULES]: jsonToString(event.modules),
     }),
-    // TODO: decide what to do with tags
+    // TODO: decide what to do with these sentry specific tags
```

Comment:
should we put some of these in otel semantic conventions properties instead?

Reply:
I couldn't find a very good mapping for theses. we can remap this after finalizing the prototype